test/src/org/labkey/test/tests/panoramapremium/TargetedMSSampleManagerIntegrationTest.java

```diff
@@ -88,17 +108,24 @@ public void testSampleTypeNavigation()
         assertElementPresent(Locator.linkWithText(s2));
         assertElementPresent(Locator.linkWithText(s3));
         clickAndWait(Locator.linkWithText(s1));
-        checker().verifyTrue("Sample link did not navigate to sample manager application",
-                getCurrentRelativeURL(false).contains(WebTestHelper.buildRelativeUrl("sampleManager", getProjectName() + "/" + Sample_Manager_Subfolder, "app")));
+        checker().fatal().verifyTrue("Sample link did not navigate to Sample Manager application",
+                getCurrentRelativeURL(false).contains(WebTestHelper.buildRelativeUrl("SampleManager", getProjectName() + "/" + Sample_Manager_Subfolder, "app")));
 
         log("Navigating back to labkey server");
         waitForElementToBeVisible(Locator.linkWithText("Assays"));
         click(Locator.linkWithText("Assays"));
         waitAndClick(Locator.linkContainingText("Skyline Documents"));
         waitAndClickAndWait(Locator.linkWithText(SProCoP_FILE_ANNOTATED));
-        checker().verifyTrue("Did not navigate back to labkey server",
+        checker().fatal().verifyTrue("Did not navigate back to labkey server",
                 getCurrentRelativeURL(false).contains(WebTestHelper.buildRelativeUrl("targetedms", getProjectName() + "/" + TargetedMS_SubFolder, "showPrecursorList")));
 
+        log("Change to the LIMS configuration to verify the navigation");
+        setProductConfigurationViaApi(ProductKey.labkeyLims);
+        clickAndWait(Locator.linkWithText("6 replicates"));
+        clickAndWait(Locator.linkWithText(s1));
+        checker().fatal().verifyTrue("Sample link did not navigate to LIMS application",
+                getCurrentRelativeURL(false).contains(WebTestHelper.buildRelativeUrl("LIMS", getProjectName() + "/" + Sample_Manager_Subfolder, "app")));
```

Comment:
Use advanced assertions library to produce more useful error messages. This would help with the current failure.
```suggestion
        Assertions.assertThat(getCurrentRelativeURL()).as("Sample link navigation for LIMS product")
            .contains(WebTestHelper.buildRelativeUrl("LIMS", getProjectName() + "/" + Sample_Manager_Subfolder, "app"));
```

Reply:
done